tests/MulticastUDPClient.py

```diff
@@ -260,8 +261,8 @@ def say(self, address, port, conn, msg):
 		"""
 		conn.sendto(bytes(msg + "\n", "utf-8"), (address, port))
 		received = str(conn.recv(1024), "utf-8")
-		print("Sent:     {}".format(msg))
-		print("Received: {}".format(received))
+		print(str("Sent:     {}").format(msg))  # skipcq: PYL-C0209  -  must remain compatible
+		print(str("Received: {}").format(received))  # skipcq: PYL-C0209  -  must remain compatible
```

Comment:
While I agree with the reasoning, this part of the code has been flaky when linted by [Deepsource](app.deepsource.com) so I marked it explicitly to use the more compatible `str().format()` syntax so it would stop pushing f-strings for now. We'll re-visit this in a future. For this PR/MR the change is to suppress pylinter C0209 noise